### ISSUE TITLE
Infinite loop on new blog

### ISSUE THREAD

**Issue by MichalBryxi** [empress/empress-blog#40]
Followed the basic installation. Created new user & post and got this infinite loop:

```
ember g user michal-bryxi
ember generate post "The fun with pounds" --author=michal-bryxi
```

Then the server spits following in the console:
```
There was an error running your app in fastboot. More info about the error:
 Error: Cannot find module 'moment' from '/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees'
    at Function.module.exports [as sync] (/Users/mbryxi/Projects/unicorn-misa-pudr-com-empress/node_modules/resolve/lib/sync.js:71:15)
    at Object.require (/Users/mbryxi/Projects/unicorn-misa-pudr-com-empress/node_modules/fastboot/src/ember-app.js:139:42)
    at Module.callback (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/moment/fastboot-moment.js:4:34)
    at Module.exports (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/assets/vendor/loader/loader.js:106:1)
    at Module._reify (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/assets/vendor/loader/loader.js:104:1)
    at Module._reify (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/var/folders/dw/bx1n948x7bgdmwdpxgns8q040000gn/T/broccoli-67747c4ty67J6SLnb/out-391-broccoli_merge_trees/assets/vendor/loader/loader.js:104:1)
2019-07-23T08:25:17.398Z 200 OK /
```

That might have been there before. The main problem is that the website in the browser will infinitely loop on following URL and try to fetch it over and over again. Did not really get more sensible data as it completely blocks my dev tools:

Screen of the post in the browser:
![Screenshot 2019-07-23 at 09 25 39](https://user-images.githubusercontent.com/847473/61695866-2e644400-ad2c-11e9-9637-401954271df6.png)

Screen of the request in the network panel. There are thousands of those after few seconds:
![Screenshot 2019-07-23 at 09 25 57](https://user-images.githubusercontent.com/847473/61695873-3328f800-ad2c-11e9-8231-6d8bcc8fdea4.png)

**Comment by MichalBryxi** [comment]
Just found out that if I open the markdown of the new post `content/the-fun-with-pounds.md` and delete lines:

```
tags:
  - new
``` 

Then this problem goes away.

**Comment by reggietheroman** [comment]
Just ran into this problem. Another way to fix it would be to create a tag named new using
```
ember g tag new
```

**Comment by mansona** [comment]
hey folks sorry for the delay in getting back to you, yes that is a tricky situation 🤔 I think there is an easy enough way to fix this for new blogs (make sure the new tag is there by default) 

This issue will also come up if you have a post that you create with any tag that hasn't previously been generated so we should probably have a check for that in the build system that will tell us exactly what is wrong 👍 

I'll be able to get to this over the weekend

**Comment by MichalBryxi** [comment]
Not sure why frontend tries to re-fetch the data all the time. Seems like a dangerous practice to me, especially when there is no exponential falloff or something.
Wouldn't it be better to simply fail and not try re-fetch over and over again?

**Comment by mansona** [comment]
I'm not entirely sure why it's doing it either 🤔 I would imagine that it's a quirk of ember-data that I don't know how to exactly reproduce independently of empress-blog but it's something that we should probably raise with them.

I have just released empress-blog@1.7.2 which fixes our issue. Firstly I have made sure that newly generated blogs have the default `new` tag available, and secondly I've added a check during build time that will stop the build if you have defined a tag on a post that doesn't exist yet 👍
